tests/test_cosserat_curve.py

```diff
@@ -251,7 +262,9 @@ def test_cosserat_curve_mesh_warp_transform_boundary_conditions():
         ),
     )
 
-    # Compare with the reference result
-    compare_test_result_pytest(
-        mesh.get_string(check_nox=False, header=False), rtol=1e-8, atol=1e-8
+    reference_string = get_string(
+        reference_file_directory / (current_test_name + ".dat")
     )
+    result_string = mesh.get_string(check_nox=False, header=False)
```

Comment:
Like I mentioned before, do we really want to do this? We would have lines that look like this in almost every test case, producing a lot of duplicated code. Maybe, if we don't want the `check_nox=False, header=False` arguments all the time, we can consider allowing an `InputFile` to be passed to `compare_results`.